docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -624,7 +709,7 @@ Now all methods and classes carrying `@HrTenant` will be authenticated using the
 OIDC provider.
 
 [[tenant-config-resolver]]
-== Programmatically Resolving Tenants Configuration
+== Dynamic Tenant configuration resolution
```

Comment:
```suggestion
== Dynamic tenant configuration resolution
```
or as per my comment about the previous section 2 heading:

```suggestion
== Configure the resolution of dynamic tenants
```

The main thing is that both headings are aligned, grammatically.